cloud/satellite.go

```diff
@@ -279,31 +312,41 @@ func (c *Client) SleepSatellite(ctx context.Context, name, orgName string) (out
 		ctxTimeout, cancel := context.WithTimeout(ctx, 5*time.Minute)
 		defer cancel()
 		defer close(out)
-		stream, err := c.compute.SleepSatellite(c.withAuth(ctxTimeout), &pb.SleepSatelliteRequest{
-			OrgId:          orgID,
-			Name:           name,
-			UpdateInterval: durationpb.New(10 * time.Second),
-		})
-		if err != nil {
-			out <- SatelliteStatusUpdate{Err: errors.Wrap(err, "failed opening satellite sleep stream")}
-			return
-		}
-		for {
-			update, err := stream.Recv()
-			if err == io.EOF {
-				return
-			}
+		for i := 1; i <= 5; i++ {
```

Comment:
It seems like there's a bunch of duplicated code here. Can we create a reusable method that accepts the client call invocation as a function parameter?

Reply:
TBH I think we should just limit the patch to the reserve call anyway. This is a bit of a weird band-aid solution that I'm not sure we need to propagate everywhere. We could leave it in the one place that is affecting users and see if it improves things first.